lib/code_corps/accounts/users.ex

```diff
@@ -0,0 +1,11 @@
+defmodule CodeCorps.Accounts.Users do
+  alias CodeCorps.ProjectUser
+
+  import Ecto.Query
+
+  def project_filter(query, %{"project_id" => project_id}) do
+    from user in query,
+      join: pu in ProjectUser, on: pu.user_id == user.id and pu.project_id == ^project_id
+  end
+  def project_filter(query, _), do: query
```

Comment:
Could we add a unit test for this `project_filter`?

Reply:
What do you you think the test should do? There is one test already in the `user_controller_test`. How would this test be different?

Reply:
@vishaldeepak  I would say they're testing two different things in isolation. True, we are testing this code via the user controller, but that behavior could change. We want a test for this specific unit of code so that even if the user controller changes, we still know this chunk of code is good... does this make sense?

Reply:
added a test case let me know what you think. @begedin

Reply:
Test unit looks fine. This ought to be good to go.